crates/typst/src/visualize/image/mod.rs

```diff
@@ -257,6 +257,10 @@ fn layout_image(
         frame.clip(Path::rect(frame.size()));
     }
 
+    if let Some(label) = elem.label() {
+        frame.set_label(label);
```

Comment:
Why also on image. Didn't we want to keep it to box and block?

Reply:
Alright, was experimenting, I will remove it.